packages/backend-common/api-report.md

```diff
@@ -853,4 +855,10 @@ export function useHotCleanup(
 
 // @public @deprecated
 export function useHotMemoize<T>(_module: NodeModule, valueFactory: () => T): T;
+
+// @public (undocumented)
+export class UserInfoCompat implements UserInfoService {
```

Comment:
We don't wanna export this directly, the compat helper is the way to get access to it for now